sympy/crypto/crypto.py

```diff
@@ -2705,7 +2723,100 @@ def decipher_elgamal(msg, key):
     return u * c2 % p
 
 
-################ Diffie-Hellman Key Exchange  #########################
+"""##########Columnar Transposition Cipher################"""
+
+
+r"""
+Columnar Transposition Cipher
+This cipher involves writing the plaintext out in rows
+and then reading the ciphertext off in columns one by one.
+"""
+
+
+def encipher_ctc(message, key):
+
+    r"""
+    Usage
+    >>> from sympy.crypto.crypto import encipher_ctc
+    >>> key = "HI"
+    >>> encipher_ctc("HelloIlovecrypto", key)
+
+    """
```

Comment:
IMO, numpydoc string guide should be followed here as it is followed in the rest of the codebase.

Reply:
Please add references to the relevant literature as well.